.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
 npm-debug.log
+.idea/
```

Comment:
Ohh didn't read the whole thing (thought you meant the global .gitignore), got'ya.